meta-facebook/gt-cc/src/platform/plat_isr.c

```diff
@@ -42,34 +47,32 @@ void dc_on_init_component()
 		pex89000_init_arg *init_arg = (pex89000_init_arg *)cfg->init_args;
 
 		/* Only need initial when not initial yet */
-		if (init_arg->is_init == false) {
+		if (!init_arg->is_init) {
 			if (cfg->pre_sensor_read_hook) {
-				if (cfg->pre_sensor_read_hook(cfg->num,
-							      cfg->pre_sensor_read_args) == false) {
-					printf("[%s] sensor 0x%x pre sensor read failed!\n",
-					       __func__, cfg->num);
+				if (!cfg->pre_sensor_read_hook(cfg->num,
+							       cfg->pre_sensor_read_args)) {
+					LOG_ERR("[%s]sensor 0x%x pre sensor read failed!", __func__,
```

Comment:
__func__ can be removed, it's implicit in the debug output depending on the setting when the image is built.